### ISSUE TITLE
How to activate sonargraph

### ISSUE THREAD

**Issue by MattSom** [SonarQubeCommunity/sonar-sonargraph#8]
Hello,

I ran sonnar-scanner in -X, and got the following message:

INFO: Sonargraph: Skipping project cppcheck_with_absolute_paths_project [cppcheck_with_absolute_paths_project], since no Sonargraph rules are activated in current SonarQube quality profile.

The thing is I just simply can't find it at the Quality profile section at SQ.

**Comment by ingmarkellner** [comment]
Hi,

I assume you have downloaded and installed the Sonargraph plugin on your SonarQube server.

As the message tries to indicate, the plugin is only active, if any of its rules are active in the current SonarQube profile.
Note that Sonargraph is only working on Java projects, so you need to check the "Java" programming language on the rules configuration page to actually see the rules.

I will try to make this more explicit in the plugin's description and I will check if I simply omit that logging info message if the analyzed project's language is not Java.

In short: This is log statement is only an info, nothing critical. If you only analyze C++ projects, you might want to uninstall the plugin.

**Comment by MattSom** [comment]
Hi,

I understand now, thank you.

**Comment by ReinoutW** [comment]
I have this same issue, but it still doesn't silently fail when your project is not Java?

Edit:
We can't uninstall the plugin since both Java and C# projects are analyzed by this same server.

**Comment by a-zitzewitz** [comment]
@ReinoutW Do you still have this problem? Currently our SonarQube plugin only supports Java and should ignore projects of other languages.
